docs/spark/tutorials/get-started.md

```diff
@@ -40,7 +40,7 @@ Before you begin, make sure you can run `dotnet`, `java`, `mvn`, `spark-shell` f
     * Use the PowerShell command `java -version` to verify the installation.
 
 4. Install [Apache Maven 3.6.0+](https://maven.apache.org/download.cgi).
-    * Download [Apache Maven 3.6.2](http://mirror.metrocast.net/apache/maven/maven-3/3.6.2/binaries/apache-maven-3.6.2-bin.zip).
+    * Download [Apache Maven 3.6.2](https://mirror.metrocast.net/apache/maven/maven-3/3.6.2/binaries/apache-maven-3.6.2-bin.zip).
```

Comment:
One last request, sorry. I was just double checking the links and this one is not working for me with https

```suggestion
    * Download [Apache Maven 3.6.2](http://mirror.metrocast.net/apache/maven/maven-3/3.6.2/binaries/apache-maven-3.6.2-bin.zip).
```

Reply:
Weird -- it definitely worked for me yesterday; even have the zip file in my downloads directory from trying it 😆 

> ![image](https://user-images.githubusercontent.com/2148318/68056050-8ebe3680-fcc8-11e9-967a-a73cec306476.png)

Will accept the suggestion 👍